gnovm/stdlibs/std/emit_event.gno

```diff
@@ -0,0 +1,19 @@
+package std
+
+// Emit is a function that constructs a gnoEvent with a specified type and attributes.
+// It then forwards this event to the event logger. Each emitted event carries metadata
+// such as the event type, the initializing realm, and the provided attributes.
+//
+// The function takes type and attribute strings. typ (or type) is an arbitrary string that represents
+// the type of the event. It plays a role of indexing what kind of event occurred. For example,
+// a name like "Foo" or "Bar" can be used to indicating the purpose (nature) of the event.
+//
+// And the attrs (attributes) accpets an even number of strings and sets them as key-value pairs
+// according to the order they are received. For example, if the attr strings "key1", "value1" are
```

Comment:
```suggestion
// according to the order they are passed. For example, if the attr strings "key1", "value1" are
```

Reply:
@notJoon 
This typo is still unresolved 🙏

Reply:
Nevermind, I cam crazy, my local git history was not synced up